cmd/zar/README.md

```diff
@@ -406,6 +406,74 @@ URI                     SUM
 ```
 Pretty cool!
 
+## simple graph queries
+
+Here is another example to illustrate the power of zar/zq.  Just like you can
+build search indexes with arbitrary zql commands, you can also build graph indexes
+to hold edge lists and node attributes, providing an efficient means to do
+graph queries.  While this doesn't provide a full-featured graph database
+like [neo4j](https://github.com/neo4j/neo4j),
+it does provide a nice way to do many types of graph queries that
+could prove useful for your archive analytics.
+
+For example, to build an searchable edge list comprising communicating IP addresses,
+run the following commands to a create a multi-key search index
+keyed by all unique instances of IP address pairs (in both directions)
+where each index includes a count of the occurrences.
+```
+zar zq -o forward.zng "id.orig_h != null | put from=id.orig_h,to=id.resp_h | count() by from,to" _
+zar zq -o reverse.zng "id.orig_h != null | put from=id.resp_h,to=id.orig_h | count() by from,to" _
+zar zq -o directed-pairs.zng "count() by from,to" forward.zng reverse.zng
+zar zdx -o graph -k from,to directed-pairs.zng
+```
+> (Note: there is a small change we can make to zql to do this with one `zar zq`
+> command... coming soon.)
+
+This creates an index called "graph" that you can use to search for IP address
+pair relationships, e.g., you can say
+```
+zar find -z -x graph 216.58.193.195 | zq -f table "count() by from,to" -
```

Comment:
```suggestion
zar find -z -x graph 216.58.193.195 | zq -f table "sum(count) as count by from,to" -
```

Reply:
oops... I'll fix the example table output too.  I should have realized something was wrong when the counts were too small!  thx